test/checks/aria/aria-level.js

```diff
@@ -0,0 +1,41 @@
+describe('aria-prohibited-attr', function() {
+  'use strict';
+
+  var checkContext = axe.testUtils.MockCheckContext();
+  var checkSetup = axe.testUtils.checkSetup;
+  var checkEvaluate = axe.testUtils.getCheckEvaluate('aria-level');
+
+  afterEach(function() {
+    checkContext.reset();
+  });
+
+  it('should return true if aria-level is 6', function() {
```

Comment:
This and the next test title are switched (or the contents are switched, one of the two)